interface/globals.php

```diff
@@ -287,6 +287,13 @@ function undoMagicQuotes($array, $topLevel=true) {
       if ($gl_value == '2') $GLOBALS['sell_non_drug_products'] = 1;
       else if ($gl_value == '3') $GLOBALS['sell_non_drug_products'] = 2;
     }
+    else if ($gl_name == 'gbl_time_zone') {
+      if ($gl_value) {
```

Comment:
Just place a quick comment here what doing here and that this value (or default php setting is not set) will be used in downstream mysql timezone settings.
Then if it tests ok, I think it's ready to put in and see what happens.